src/node/desktop/src/ui/widgets/choose-r/ui.html

```diff
@@ -1,37 +1,37 @@
 <!DOCTYPE html>
 <html>
```

Comment:
I think this would be more suited for the task to decide the language for the electron app don't you think? As we don't have this feature right now, we should maybe concentrate those changes in the scope for that other task. Let me know your take on it

Reply:
Absolutely, it can wait for that.